api/logs/views.py

```diff
@@ -0,0 +1,216 @@
+from modularodm import Q
+from rest_framework import generics, permissions as drf_permissions
+from rest_framework.exceptions import NotFound
+
+from website.models import Node, NodeLog
+
+from framework.auth.oauth_scopes import CoreScopes
+
+from api.base.filters import ODMFilterMixin
+from api.base.utils import get_user_auth
+from api.base import permissions as base_permissions
+from api.logs.serializers import NodeLogSerializer
+from api.nodes.serializers import NodeSerializer
+from api.nodes.utils import get_visible_nodes_for_user
+
+class LogList(generics.ListAPIView, ODMFilterMixin):
```

Comment:
As I dig in more, I see what you're saying about using the get_aggregate_logs_queryset. That being said, a quick refactor will allow us to grab the query rather than the queryset, thus allowing us to filter with mongo rather than the list filter. So probably worth doing.